src/zeit/content/image/imagegroup.py

```diff
@@ -192,6 +197,17 @@ def get_variant_fill(self, key):
     def get_variant_viewport(self, key):
         return get_viewport_from_key(key)
 
+    def get_scale(self, key):
+        """If key contains `2x`, retrieve 2 else None"""
+        for segment in key.split('__')[1:]:
+            seg = segment.split('x')
+            if len(seg) is 2 and not seg[1]:
```

Comment:
Nur nebenbei (das erledigt sich mit obigem von selber), `is` ist für Objektvergleiche, das ist hier das Falsche.